### PR TITLE
Feature/flickbook demo

### PR DESCRIPTION
**Pull request by tomcartwrightuk**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -9,11 +9,6 @@ var buildPath = './demo/build/';
 
 gulp.task('auto-reload', loadLmnTask('auto-reload'));
 
-gulp.task('html', loadLmnTask('html', {
-  langBase: 'component.flick',
-  imagePath: '../../src/imgs/'
-}));
-
```
Comment (callumacrae) [inline gulpfile.js:16]: This is used

File: package.json [hunk 2]
```diff
@@ -7,7 +7,8 @@
   "dependencies": {
     "after-delay": "^1.0.0",
     "browserify": "^6.2.0",
-    "jquery": "^1.11.2"
+    "jquery": "^1.11.2",
+    "lmn-gulp-tasks": "^1.3.4"
```
Comment (callumacrae) [inline package.json]: Sorry, this should have been a devDependency

File: src/js/helpers/isMobile.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+/**
+ * Returns whether browser is a mobile or not. Tests for touch support and
+ * screen width.
+ *
+ * @returns {boolean} True if mobile.
+ */
+module.exports = function () {
+  // If doesn't support touch
+  if (!('ontouchstart' in window) && !navigator.msMaxTouchPoints) {
```
Comment (callumacrae) [inline src/js/helpers/isMobile.js]: Use https://www.npmjs.com/package/touch-screen. Where did you take this from? I'll update it there, too.

File: src/js/flick.js [hunk 4]
```diff
@@ -11,25 +12,19 @@ $.flick.inputs = {
...
-  input: 'tilt' // String or function
-};
-
 /**
  * Make a flicky thing!
  *
  * @param {object} options Object containing some options.
  */
 $.fn.flick = function flick(options) {
-  options = $.extend({}, $.flick.defaults, options);
```
Comment (callumacrae) [inline src/js/flick.js:24]: This line should be kept in. Have the default `input` be "auto", and set it to "mouseX" or "tilt" if it's "auto". Lot DRYer and more extensible :)

Comment (callumacrae) [inline src/js/flick.js:24]: Basically just the old code plus this:

```
if (options.input === 'auto') {
    options.input = isMobile() ? 'flick' : 'mouseX';
}
```

File: gulpfile.js (hunk 1 above)
Comment (tomcartwrightuk) [inline gulpfile.js:16]: How do I stop it butchering the urls in the partial? It is mkaing the urls look like this:
`../..imgs/letters/http://img.x/ala...` etc

Comment (callumacrae) [inline gulpfile.js:16]: update to `lmn-gulp-tasks@1.3.5`, there was a bug in `rubbish-erb-parser` in an older version and I forgot to update lmn-gulp-tasks.

**Comment by callumacrae** [comment]
dude! you just removed the html task, which will break the project for everyone but you.

also isMobile.js is unused

The actual code looks good, though :smile:

**Comment by tomcartwrightuk** [comment]
Ah yes - forgot to return that.

**Pull request merged**